docs/integrations/development.md

```diff
@@ -36,25 +36,25 @@ required for authenticating with provider APIs and otherwise necessary to
 configure the integration for execution. This varies between services.
 
 The `type` will ensure the values are cast when
-[read from `.env`](#j1-integration-collect). It is important to mark secrets
-with `mask: true` to facilitate safe logging.
+[read from `.env`](#command-j1-integration-collect). It is important to mark
+secrets with `mask: true` to facilitate safe logging.
 
 Example:
 
 ```typescript
 {
   clientId: {
-    type: 'string',
-    mask: false,
-  },
+    type: 'string';
+    mask: false;
```

Comment:
This is `instanceConfigFields` not a type definition, so I think the `,` are correct.

Reply:
Good catch! I'll take another look at this in the release PR. My editor was complaining about it and these changes made the complaints go away. 😬